### ISSUE TITLE
How will this help you degenerates?

### ISSUE THREAD

**Issue by raitonoberu** [stop-war-in-ukraine/stop-russia-it#41]
Эта «інформаційна війна» сейчас с нами в одной комнате?

**Comment by jsisaev1** [comment]
@raitonoberu нет. они истерят по всему западному интернету и говорят, что русским надо отрезать весь интернет, все сервисы. только эти дегенераты не понимают, что во время самых жестких санкций в СССР так или иначе присутствовали  санкционные машины типа IBM 360/390, dec pdp-11 и другие, а также были клоны доса и юникса. когда-то даже интернет (году в 1985) появился. 

эти истерички,  в особенности, соевый iimetra истерично отклоняет PR, в которых я писал правду про происходящее в/на Украине и в украинском ИТ в частности.

так что все, кто видит и может прочитать этот комментарий полностью, до этих слов, не сломав устройство от злости, с которого читатель видит эти символы, подавайте жалобы на пользователя iimetra и stop-war-in-ukraine . гитхаб и другие площадки не должны использоваться для координации травли по какому бы ни было признаку.

**Comment by greatbody** [comment]
We love America even if it bombs civilians everywhere;
We condemn Russia even if its response is forced by America.

**Comment by digiom-ceo** [comment]
> We love America even if it bombs civilians everywhere; We condemn Russia even if its response is forced by America.

Then don't whine. We are Russian and will not let America kill us. We will kill her and her terrorist assets. Bin Laden will look like a little girl to America compared to us.

**Comment by oleksiizabara** [comment]
> так что все, кто видит и может прочитать этот комментарий полностью

нам как IT-специалистам в Украине это никак не поможет! и никак не навредит! ты прав! это просто докажет вам насколько вы бесполезны без всего, что вам предоставляет ненавистный вами запад! докажет насколько вы презираемы цивилизованным миром за поддержку фашистского режима на россии! 

сейчас не 1985! проснись! отрезать россию и западных клиентов, которые пока еще нанимают российские галеры от  западных сервисов (даже от их малой части) - и российской IT индустрии конец! и ты сам это прекрасно понимаешь раз открываешь в горячке ПР и рассказываешь из россиянского запопинска !украинцам! как сейчас в Украине! и что происходит в Украине! 

и вообще! главнокомандующий дал указание на импртозамещение! вот и поддержи отечественных производителей ПО или девайсов! а! стоп! некого же поддерживать! все хорошее, что вы когда-то сделали уже давно не на россии, потому что умные люди давно поняли, что на россии нельзя честно вести бизнес! 

так что останется на россии какая-то сотня другая невыездных писать в бункерах ПО для ваших ракет и танков, а остальным добро пожаловать продолжать карьеру в Украине! мы будем рады видеть вас в нашей дружной стране!

**Comment by exceedev** [comment]
> > так что все, кто видит и может прочитать этот комментарий полностью
> 
> нам как IT-специалистам в Украине это никак не поможет! и никак не навредит! ты прав! это просто докажет вам насколько вы бесполезны без всего, что вам предоставляет ненавистный вами запад! докажет насколько вы презираемы цивилизованным миром за поддержку фашистского режима на россии!
> 
> сейчас не 1985! проснись! отрезать россию и западных клиентов, которые пока еще нанимают российские галеры от западных сервисов (даже от их малой части) - и российской IT индустрии конец! и ты сам это прекрасно понимаешь раз открываешь в горячке ПР и рассказываешь из россиянского запопинска !украинцам! как сейчас в Украине! и что происходит в Украине!
> 
> и вообще! главнокомандующий дал указание на импртозамещение! вот и поддержи отечественных производителей ПО или девайсов! а! стоп! некого же поддерживать! все хорошее, что вы когда-то сделали уже давно не на россии, потому что умные люди давно поняли, что на россии нельзя честно вести бизнес!
> 
> так что останется на россии какая-то сотня другая невыездных писать в бункерах ПО для ваших ракет и танков, а остальным добро пожаловать продолжать карьеру в Украине! мы будем рады видеть вас в нашей дружной стране!

выйди в окно, прежде чем подобное писать. а да и выучи что нибудь кроме хтмл и сиэсекс))))))

**Comment by oleksiizabara** [comment]
> выйди в окно, прежде чем подобное писать. а да и выучи что нибудь кроме хтмл и сиэсекс))))))

во-первых я не пишу на html и css и даже не на js
то что ты видишь репозитории, которые созданы четыре года назад для детей в образовательных целях - это тебе об этом сказало? убогий! или ты думал увидеть в паблике коммерческие проекты?

можешь найти мой профиль на линке и узнать чем я занимаюсь! если он у вас еще не закрыт

во-вторых я и так все вижу в окно! а твои уши засраны говном от соловьева и захаровой!

спасай свою страну! как мы спасаем нашу! ваши немощные комментарии здесь неуместны - только доказательство того как печет чертям святая вода!
